scene/gui/file_dialog.h

```diff
@@ -79,6 +79,7 @@ class FileDialog : public ConfirmationDialog {
 	OptionButton *drives;
 	Tree *tree;
 	LineEdit *file;
+	Label *fileLabel;
```

Comment:
We use `snake_case` for identifiers.

Reply:
Fixed and squashed together into one commit